### ISSUE TITLE
Add git support

### ISSUE THREAD

**Issue by Philipp-M** [Unity-CI/docker#40]
**Bug description**
I'm getting the following error using the unity-builder github action 2.0-alpha:

```
An error occurred while resolving packages:
  Project has invalid dependencies:
    net.gree.unity-webview: No 'git' executable was found. Please install Git on your system then restart Unity and Unity Hub
```

**How to reproduce**
Use a dependency in the manifest.json that needs git, e.g. this:
https://github.com/gree/unity-webview/#package-manager

**Expected behavior**
All git dependencies are resolved correctly

**Comment by webbertakken** [comment]
@Philipp-M thanks a lot for raising this issue!

We have already discussed this on [discord](https://unity-ci.com/discord) and decided we will add both Git and Git-LFS to all images, as they're considered core dependencies, for the reason you're describing also.

**Comment by webbertakken** [comment]
Released in [v0.5](https://github.com/Unity-CI/docker/releases/tag/v0.5)
